src/io/flutter/view/TogglePlatformAction.java

```diff
@@ -55,7 +55,13 @@ public final void update(AnActionEvent e) {
       }
 
       final int platformIndex = extensionDescription.getValues().indexOf(selectedPlatform.name());
-      selectorText = (String)extensionDescription.getTooltips().get(platformIndex);
+      try {
+        selectorText = (String)extensionDescription.getTooltips().get(platformIndex);
+      }
+      catch (ArrayIndexOutOfBoundsException ex) {
+        selectorText = "Platform: Unknown";
+        System.out.println("Unknown platform: " + selectedPlatform.name());
+      }
```

Comment:
Alternatively, maybe just check for `-1` rather than trapping the exception? 

Dunno if that's much better...  🤷‍♂

Reply:
done.